common/flags/Flags.java

```diff
@@ -104,7 +104,13 @@ static void setFlagValue(String name, String value) {
 
   @VisibleForTesting
   static String getDefaultFlagValue(String flagName) {
-    return instance().getFlag(flagName).getDefault();
+    FlagData flagData = instance().getFlag(flagName);
+    if(flagData.getIsListFlag()){
+      // deleting characters "[" and "]" at the beginning and end of the List
```

Comment:
Why would we have `"[" and "]"` at the beginning and end of the list?

Reply:
We have these symbols from a default value of the flag of a list. 
When we create a new flag of the list:
```
@FlagDesc(name = "strings_list_flag", description = "A flag description")
  public static final Flag<List<String>> stringsListFlag
      = Flag.createStringsListFlag(Arrays.asList("ab", "cd"));
```
And we want to get a default value we will get `[ab, cd]` because `toString()` method is called. This call happens here - https://github.com/google/startup-os/blob/4aa5dcff4545d276190a8501b58ec8666f3109d8/common/flags/ClassScanner.java#L188